app/controllers/api/payment/braintree_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Api::Payment::BraintreeController < ApplicationController
```

Comment:
we're going to end up undoing part of this namespacing, but only part of it cause I think we'll leave the webhooks in here.